<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NurseDetailsWindow</class>
 <widget class="QMainWindow" name="NurseDetailsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Detalhes do Enfermeiro</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>331</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 238, 236);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLineEdit" name="cpfTag">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>41</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>CPF:</string>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="nameTag">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>60</y>
       <width>51</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Nome:</string>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="cpfLineEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>191</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="nameLineEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>191</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="bornTag">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>141</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Data de nascimento:</string>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="susNumberTag">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>140</y>
       <width>121</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Registro COREN:</string>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="numberCorenlineEdit">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>140</y>
       <width>151</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="removeNurseButton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>190</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Remover enfermeiro</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="updateNurseButton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>240</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Atualizar enfermeiro</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="birthDateLineEdit">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>100</y>
       <width>131</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCommandLinkButton" name="backButton">
     <property name="geometry">
      <rect>
       <x>131</x>
       <y>290</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>voltar para gerenciamento</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
